server/controllers/entities/lib/special_entity_images_getter.js

```diff
@@ -21,6 +21,12 @@ module.exports = {
     const worksUris = _.map(parts, 'uri')
     const worksImages = await Promise.all(worksUris.map(getOneWorkImagePerLang))
     return worksImages.reduce(aggregateWorkImages, images)
+  },
+
+  collection: async entity => {
+    entity.images = {}
+    return entities_.byClaim('wdt:P195', entity.uri, true, true)
+    .then(addEditionsImages(entity.images))
```

Comment:
```suggestion
    const images = { claims: getEntityImagesFromClaims(entity) }
    return entities_.byClaim('wdt:P195', entity.uri, true, true)
    .then(addEditionsImages(images))
```
`getEntityImagesFromClaims` allows to make images from Wikidata available on the  returned `images` object, which could provide good illustrations for collections old enough to be in the public domain, for instance.